antispam.py

```diff
@@ -0,0 +1,54 @@
+import time
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class AntiSpam:
+    _count = None
+    _timeout = None
+    _lookup = {}
+
+    def __init__(self, count=2, timeout=10) -> None:
```

Comment:
a description about what each parameter means would be great. Like user is allowed to execute `count` commands during `timeout` time or so